src/base/utils/fs.cpp

```diff
@@ -238,40 +239,10 @@ qulonglong Utils::Fs::freeDiskSpaceOnPath(const QString &path)
 {
```

Comment:
oops, replied too fast, `friendlyUnit(-1)` will return translated string: `Unknown`, this is more preferable,
don't you think?